### DISCUSSION TITLE
GridLayout多个nodes报错

### DISCUSSION THREAD

**Discussion by 1476339526** [antvis/X6#2454] [Q&A]
![微信截图_20220802175704](https://user-images.githubusercontent.com/38809324/182347641-2819592c-5947-494f-a5e7-2a16fa97ed64.png)

如上图，当GridLayout的nodes含多条数据报错“n is not a function or its return value is not iterable”。
我将代码移至在线能渲染能成功，但是在我的环境中无法编译。
目前我不知道是我的版本问题，还是其他，无法定位。

**Comment by NewByVector** [comment]
可以提供一个可以复现的 codesanbox 吗？
